modules/nf-core/umicollapse/main.nf

```diff
@@ -28,6 +29,10 @@ process UMICOLLAPSE {
     // which leaves 5% for stuff happening outside of java without the scheduler killing the process.
     def max_heap_size_mega = (task.memory.toMega() * 0.9).intValue()
     def max_stack_size_mega = (task.memory.toMega() * 0.05).intValue()
+    def file_type = args.contains("fastq") ? "fastq" :
+                    args.contains("bam") ? "bam" :
+                    input.getExtension()
```

Comment:
Might this end up with `gz` as the extension?

Reply:
Yes, good catch! I will remove the third clause.